src/logger.js

```diff
@@ -86,7 +86,7 @@ function infoLog(value) {
  * DEBUG:: VALUE
  */
 function debugLog(value) {
-  if (debug) {
+  if (config.debug()) {
```

Comment:
```suggestion
  if (debug) {
```
Then again here we would just have it as a regular variable.